tool/prepare-release.ts

```diff
@@ -50,6 +50,14 @@ async function sanityCheckBeforeRelease() {
     );
   }
 
+  for (const [dep, version] of Object.entries(pkg.optionalDependencies)) {
+    if (version !== releaseVersion) {
+      throw Error(
+        `optional dependency ${dep}'s version has not been updated to ${releaseVersion}.`
```

Comment:
```suggestion
        `optional dependency ${dep}'s version doesn't match ${releaseVersion}.`
```

I think it's clearer to just describe the problem here rather than try to explain its etiology.